sonarqube/chart/templates/secrets-postgresql.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
+  database-password: {{ .Values.global.sonarDatabasePasswordB64 }}
+kind: Secret
+metadata:
+  annotations:
+    template.openshift.io/expose-database_name: '{.data[''database-name'']}'
+    template.openshift.io/expose-password: '{.data[''database-password'']}'
+    template.openshift.io/expose-username: '{.data[''database-user'']}'
```

Comment:
Are those annotations needed / correct? It looks like the username and DB name are in the configmap?

Reply:
True, none of this annotations are needed. Will remove